Doc/c-api/sys.rst

```diff
@@ -335,6 +335,15 @@ accessible to C code.  They all work with the current interpreter thread's
    .. versionadded:: 3.8
 
 
+.. c:function:: void _PySys_ClearAuditHooks()
+
+   Clears all audit hooks. The function is an internal implementation detail
+   and only mentioned here to document the associated auditing event.
```

Comment:
We could also document the event as part of `Py_FinalizeEx`, rather than documenting an inaccessible function (it shouldn't even be exported, let alone be used)